MANIFEST.in

```diff
@@ -1,3 +1,14 @@
+prune .github
 include LICENSE
 include README.md
 include pyproject.toml
+include *.ipynb
+include *.md
+include *.png
+include *.yaml
+include *.yml
+include .flake8
+include .coveragerc
+include Dockerfile
+recursive-include requirements *.yml
+recursive-include stratify *.pyx
```

Comment:
@HGWright This is my suggestion:
```shell
prune .github
exclude .cirrus.yml
exclude .coveragerc
exclude .flake8
exclude .gitignore
exclude .pre-commit-config.yaml
exclude CONTRIBUTING.md
exclude Dockerfile
exclude index.ipynb
include summary.png
recursive-include requirements *.yml *.lock
recursive-include stratify *.pyx
```
What do you think?

Reply:
Looks good to me @bjlittle. I have updated the file with the changes.